src/export/ValueSetExporter.ts

```diff
@@ -0,0 +1,104 @@
+import { ValueSet, ValueSetCompose } from '../fhirtypes';
+import { FSHTank } from '../import/FSHTank';
+import { FHIRDefinitions } from '../fhirdefs/FHIRDefinitions';
+import {
+  FshValueSet,
+  ValueSetComponent,
+  ValueSetConceptComponent,
+  ValueSetFilterComponent,
+  FshCode
+} from '../fshtypes';
+import { logger } from '../utils/FSHLogger';
+import { ValueSetComposeError } from '../errors';
+
+export class ValueSetExporter {
+  public readonly valueSets: ValueSet[] = [];
+  constructor(public readonly FHIRDefs: FHIRDefinitions, public readonly tank: FSHTank) {}
```

Comment:
Does the `ValueSetExporter` actually need the `FHIRDefs`? I don't see if they get used here. Doesn't hurt much to leave them though if you can see them being needed in the future.

Reply:
You're absolutely right. Those definitions never get used, so there's no reason to have them in here.